init.sh

```diff
@@ -29,6 +29,10 @@ IPAddress=$IPLoadBalancer
 
 HOSTS_LINE="$IPAddress\t$IPName"
 
+if [[ -z "$IPLoadBalancer" ]]; then
```

Comment:
When this is verified in an Ubuntu machine "[[ " is not recognized with "_init.sh: 32: init.sh: [[: not found_
" error. We need to pass "[[" only when the "if" with multiple conditions needs to be used. Hence could you please verify with single bracket ("[")?